assets/src/modules/LizmapMap.js

```diff
@@ -38,11 +40,29 @@ export default class LizmapMap {
 
```

Comment:
No initial set of _zoom from the configuration?

Reply:
I have some improvement to make about configuration on the whole code. I'll make it in another commit.